application/classes/Ushahidi/Validator/Contact/Update.php

```diff
@@ -69,18 +69,8 @@ public function valid_contact($contact, $data, $validation)
 		}
 
 		// Valid Phone?
-		// ++TODO: There's no easy to validate international numbers
-		// so just look for numbers only.
-		else if ( isset($data['type']) AND
-			$data['type'] == Contact::PHONE )
-		{
-			// Remove all non-digit characters from the number
-			$number = preg_replace('/\D+/', '', $contact);
-
-			if (strlen($number) == 0)
-			{
-				$validation->error('contact', 'invalid_phone', [$contact]);
-			}
-		}
+		// There's no easy to validate phone numbers. Even just accepting numbers
```

Comment:
@jasonmule moved this over from msheria too. We hit an issue where SMSSync was forwarding messages with a phone number like "Safaricom" .. however this probably removes the validation we wanted creating contacts from the UI.. Any ideas for a solution to both?